translations/LICENSE_zh-CN

```diff
@@ -1,11 +1,13 @@
 GLWT（祝你好运）公共许可证
-版权所有（C）每个人，除了作者
+版权所有（C）每个人，包括作者
 
 任何人都被允许复制、分发、修改、合并、销售、出版、再授权或
 任何其它操作，但风险自负。
 
 作者对这个项目中的代码一无所知。
-代码处于可用或不可用状态，没有第三种情况。
+代码处于可用或不可用之间的量子波动状态，
+如果你想使用——
```

Comment:
This line translates into "if you willing to use..." Which is absent from the original license document.